pyworkflow/utils/path.py

```diff
@@ -154,6 +155,25 @@ def makePath(*paths):
             os.makedirs(p)
 
 
+def makeTmpPath(protocol):
+    """
+    Create the scratch folder if SCIPION_SCRATCH variable is defined into the
+    Scipion config, i.o.c create tmp folder
+    """
+    tmpPath = protocol._getTmpPath()
+    if not os.path.exists(tmpPath) and len(tmpPath):
+        scratchPath = Config.SCIPION_SCRATCH
+
+        if scratchPath is None:  # Case when SCIPION_SCRATCH isn't exist. TMP folder is created
+            os.makedirs(tmpPath)
+        else:
+            folderId = protocol.getProject().getProtWorkingDir(protocol)
+            tmpScratchFolder = os.path.join(scratchPath, folderId)
```

Comment:
A couples of projects having the same run name will share the scratch folder. I think it's risky.

Maybe something like `tmpScratchFolder = os.path.join(scratchPath, projectName, folderId)` is more safety.

Reply:
Thanks @dmaluenda , actually we wanted to do that. Somehow we forgot.

Reply:
Yeah, I think it is not my genuine idea. I remember I read it somewhere XD